src/reporters/console/console-reporter.js

```diff
@@ -157,6 +157,11 @@ export default class ConsoleReporter extends BaseReporter {
       const peakMemory = (this.peakMemory / 1024 / 1024).toFixed(2);
       msg += ` Peak memory usage ${peakMemory}MB.`;
     }
+
+    if (showNetworkUsage) {
+      this.log(this._prependEmoji(`${this.getTotalNetworkUsage()} transferred`, '⬇️'));
```

Comment:
Don't forget to move this to `en.js` and use `reporter.message` for translations.